user-interface/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/update/ExperimentUpdateDialog.java

```diff
@@ -185,88 +205,4 @@ public Optional<ExperimentDraft> getOldDraft() {
       return Optional.ofNullable(oldDraft);
     }
   }
-
-  public static class ExperimentDraft implements Serializable {
-
-    @Serial
-    private static final long serialVersionUID = 5584396740927480418L;
-
-    private String experimentName;
-    private final List<Species> species;
-    private final List<Specimen> specimen;
-    private final List<Analyte> analytes;
-
-    public ExperimentDraft() {
-      species = new ArrayList<>();
-      specimen = new ArrayList<>();
-      analytes = new ArrayList<>();
-    }
-
-    public String getExperimentName() {
-      return experimentName;
-    }
-
-    public void setExperimentName(String experimentName) {
-      this.experimentName = experimentName;
-    }
-
-    public List<Species> getSpecies() {
-      return new ArrayList<>(species);
-    }
-
-    public void setSpecies(Collection<Species> species) {
-      this.species.clear();
-      this.species.addAll(species);
-    }
-
-    public List<Specimen> getSpecimens() {
-      return new ArrayList<>(specimen);
-    }
-
-    public void setSpecimens(Collection<Specimen> specimen) {
-      this.specimen.clear();
-      this.specimen.addAll(specimen);
-    }
-
-    public List<Analyte> getAnalytes() {
-      return new ArrayList<>(analytes);
-    }
-
-    public void setAnalytes(Collection<Analyte> analytes) {
-      this.analytes.clear();
-      this.analytes.addAll(analytes);
-    }
-
-    @Override
-    public boolean equals(Object object) {
-      if (this == object) {
-        return true;
-      }
-      if (object == null || getClass() != object.getClass()) {
-        return false;
-      }
-
-      ExperimentDraft that = (ExperimentDraft) object;
-
-      if (!Objects.equals(experimentName, that.experimentName)) {
-        return false;
-      }
-      if (!species.equals(that.species)) {
-        return false;
-      }
-      if (!specimen.equals(that.specimen)) {
-        return false;
-      }
-      return analytes.equals(that.analytes);
-    }
-
-    @Override
-    public int hashCode() {
-      int result = experimentName != null ? experimentName.hashCode() : 0;
-      result = 31 * result + species.hashCode();
-      result = 31 * result + specimen.hashCode();
-      result = 31 * result + analytes.hashCode();
-      return result;
-    }
-  }
```

Comment:
Why did you remove this ExperimentDraft object?

Reply:
persuaded, I reintroduced it